virtcontainers/qemu.go

```diff
@@ -658,6 +660,14 @@ func (q *qemu) stopSandbox() error {
 
 	err = q.qmpMonitorCh.qmp.ExecuteQuit(q.qmpMonitorCh.ctx)
 	if err != nil {
+		for i := 0; i < 100; i++ {
+			if q.isShutdown {
```

Comment:
It is not wrong but looks a bit racy  if this start to be used in other places. 

I wonder if you try to check if  qemu process exist  or the a qemu socket (that is known exist) is a better way to identify if VM is running or not.

Reply:
Make sense, if we do this, we can remove the logic about log qmp event

Reply:
@Ace-Tang I think its better if can check if the qmp.disconnectedCh is closed(indicating conneection has been lost) and check that the qmp.sock no longer exits to verify that the qemu instance has exited on receiving an error on `qmp.ExecuteQuit`. I dont think you can rely on SHUTDOWN msg, esp if you execute a `kill -9` on the qemu process. 

@markdryan What do you think?